src/config/dynamic-theme-fixes.config

```diff
@@ -2780,6 +2780,7 @@ body {
 mail.google.com
 
 INVERT
+tr.hR
```

Comment:
Wait NVM remove this. Some have special colors and this will fuck it up.

Reply:
will do